docs/pages/Cardinal/API-Reference/Systems.mdx

```diff
@@ -68,7 +68,7 @@ func AttackSystem(world *cardinal.World, queue *cardinal.TransactionQueue, logge
 		atk := tx.Value()
 
 		// update the component based on the attack
-		err := HealthComponent.Update(world, storage.EntityID(atk.TargetPlayer), func(health Health) Health {
+		err := HealthComponent.Update(world, entity.ID(atk.TargetPlayer), func(health Health) Health {
```

Comment:
noting here: wonder if we may need a more complex workflow - i.e. docs should be rebuilt/republished to the web *after* a tagged release, so its always always in sync with changes, since now - the only way these docs make sense is if someone pulls cardinal from main